README.md

```diff
@@ -60,6 +60,7 @@ end
 
 1. Fork it ( http://github.com/jekyll/jekyll-compose/fork )
 2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+3. Run the specs (`script/test`)
```

Comment:
I figured test was for devs and cibuild was for travis. Seems fine to leave those separate to me, but you want me to change it?